design_activity.md

```diff
@@ -10,19 +10,20 @@ While we encourage you to use the provided design for this project, there is muc
 - Compare and contrast different designs for the same program
 - Implement a program from a provided UML diagram
 
-## Your Design
+## First Draft Design
 
-Read through the project requirements with your partner, then follow our standard [object-oriented design process](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/02-intermediate-ruby/oo-design.md).
+1. Read through the project requirements with your partner, then follow our standard [object-oriented design process](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/02-intermediate-ruby/oo-design.md).
+    - What are the nouns, and how are they composed into objects and relations?
+    - What are the verbs, and which verb belongs with which object?
+2. From above, we are leading you to certain classes and dependencies. We want `lib/slack.rb` to have only **one dependency**: `Workspace`. We want `Workspace` to have at least two dependencies: `User` and `Channel`. We want `Workspace` to have many `User`s (composition). Also, we want `Workspace` to have many `Channel`s (composition). After reading through the requirements, what verbs are associated with `User`? What verbs are associated with `Channel`? What's **similar**? What's **different**?
```

Comment:
```suggestion
2. From above, we are leading you to certain classes and dependencies. We want `lib/slack.rb` to have only **one dependency**: `Workspace`. We want `Workspace` to have at least two dependencies: `User` and `Channel`. We want `Workspace` to have many `User`s (composition). Also, we want `Workspace` to have many `Channel`s (composition). After reading through the requirements:
  - What verbs are associated with `User`? 
  - What verbs are associated with `Channel`? 
  - What's **similar**? 
  - What's **different**?
```

Reply:
As a note for the future:

For GitHub Markdown formatting,

if you're mixing list style types (numbered vs. bulleted)
You need to double indent in the nested list.

I'm fixing this in a commit now but in the future that's what'll make it look good!